### ISSUE TITLE
snap으로 설치한 krita 한글 입력 안 됨

### ISSUE THREAD

**Issue by hellojack2008** [Riey/kime#346]
.xprofile에 아래 내용 작성하고,
export GTK_IM_MODULE=kime
export QT_IM_MODULE=kime
export XMODIFIERS=@im=kime

kime-xim, kime-display 실행한 kde xorg 입니다.

krita snap 한글 입력이 안 돼요

**Comment by Riey** [comment]
snap은 통상적인 ime랑은 통신이 불가능합니다 오로지 ibus로 하드코딩 돼있어 kime으로선 지원이 곤란하네요

**Comment by Riey** [comment]
@nakoo 

* Is other Qt app works?
* See krita's open files with lsof or task manager and check it loads system libraries and kime plugin

**Comment by nakoo** [comment]
> * Are other Qt apps work?
> * See krita's open files with lsof or task manager and check it loads system libraries and kime plugin

No. Due to that reason, I made a new issue. (https://github.com/Riey/kime/issues/500)
You can check the log from there.

**Comment by Riey** [comment]
오래된 이슈 정리 중입니다. 이 문제는 snap 샌드박스 구조상 kime 쪽에서 단독으로 해결하기 어렵습니다. snap은 자체 Qt/GTK 스택을 번들하기 때문에 호스트에 설치된 kime immodule이 샌드박스 안에서는 보이지 않고, snapd의 데스크톱 통합은 ibus와 fcitx만 연결해 줍니다.

다만 2021년 당시보다는 상황이 나아졌습니다. KDE Plasma나 sway처럼 `zwp_input_method_v2`를 지원하는 Wayland 컴포지터에서는 컴포지터의 text-input 프로토콜을 통해 입력이 전달되므로, snap/flatpak 앱에서도 kime이 그대로 동작합니다. 남은 문제는 X11 세션과 GNOME입니다.

근본 해결책인 ibus 인터페이스 구현은 #422 에서 추적하고 있고 이 이슈도 거기에 이미 언급돼 있으므로, 중복으로 보고 이 이슈는 닫겠습니다. 논의는 #422에서 이어가 주세요. 감사합니다.
